.github/workflows/all.yml

```diff
@@ -0,0 +1,321 @@
+          go get -d github.com/mh-cbon/go-bin-rpm \
+              && (cd "$GOPATH/src/github.com/mh-cbon/go-bin-rpm" \
+              && glide install \
+              && go install)
+          sudo apt-get update -y
+          sudo apt-get install -y fakeroot rpm
+      - name: Build
+        run: |
+          go version
+          ./build-release.sh
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: binaries
+          path: dist/
+
+  publish-docker:
+    runs-on: ubuntu-latest
+    needs: [deps, lint, test, test-cov, configure]
+    env:
+      VERSION: ${{ needs.configure.outputs.version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build
+        run: docker build -t $DOCKER_IMAGE_ID .
+      - name: Check
+        run: |
+            docker run $DOCKER_IMAGE_ID version
+            docker run $DOCKER_IMAGE_ID --help
+            docker run $DOCKER_IMAGE_ID help
+            docker run $DOCKER_IMAGE_ID run --help
+            docker run $DOCKER_IMAGE_ID inspect --help
+            docker run $DOCKER_IMAGE_ID status --help
+            docker run $DOCKER_IMAGE_ID stats --help
+            docker run $DOCKER_IMAGE_ID scale --help
+            docker run $DOCKER_IMAGE_ID pause --help
+            docker run $DOCKER_IMAGE_ID resume --help
+      - name: Publish
+        if: github.event_name != 'pull_request'
+        run: |
+          echo "REF=${{ github.ref }}"
+          echo "DOCKER_IMAGE_ID=$DOCKER_IMAGE_ID"
+          # Log into registry
+          echo "${{ secrets.DOCKER_PASS }}" | docker login -u "${{ secrets.DOCKER_USER }}" --password-stdin
+          VERSION="${VERSION#v}"
+          echo "VERSION=$VERSION"
+          docker tag "$DOCKER_IMAGE_ID" "$DOCKER_IMAGE_ID:$VERSION"
+          docker push "$DOCKER_IMAGE_ID:$VERSION"
+          # We also want to tag the latest stable version as latest
+          if [[ "$VERSION" != "master" ]] && [[ ! "$VERSION" =~ (RC|rc) ]]; then
+            docker tag "$DOCKER_IMAGE_ID" "$DOCKER_IMAGE_ID:latest"
+            docker push "$DOCKER_IMAGE_ID:latest"
+          fi
+
+  publish-github:
+    runs-on: ubuntu-latest
+    needs: [deps, lint, test, test-cov, configure, build]
+    if: startsWith(github.ref, 'refs/tags/v')
+    env:
+      VERSION: ${{ needs.configure.outputs.version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Download binaries
+        uses: actions/download-artifact@v2
+        with:
+          name: binaries
+          path: dist
+      - name: Create release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -x
+          assets=()
+          for asset in ./dist/*; do
+            assets+=("-a" "$asset")
+          done
+          hub release create "${assets[@]}" -m "$VERSION" -m "$(cat ./release\ notes/${VERSION}.md)" "$VERSION"
+      # - name: Upload packages to Bintray
```

Comment:
I assume you had these commented out while you were experimenting in your own repo? We can uncomment them now, right?

Reply:
These commands actually haven't been tested yet, as we don't have a test repository in Bintray and I don't want to accidentally publish something... I added the secrets in the repo and am fairly confident they'll work, but I'm apprehensive about testing them during an actual release. Any ideas how we could test them? Maybe setup a mock Bintray server and upload there?

Reply:
hmm... bintray have an open-source tier: https://bintray.com/signup/oss

can't we make another free account like that and add its credentials to your k6 fork, so you can fully test everything, including the artifact upload? after all, your k6 fork is still open-source, so it should count, we'd just be using the account for testing purposes (which should be publicly mentioned in the description, so some k6 users don't stumble on it by mistake from Google or something)

Reply:
Hhmm OK, let me see if I can set that up.

Reply:
I am still fo the opinion that we can test on release ... if it works - great. If it doesn't you will need to the more or less the same thing as you do now to fix it but without half the changing of stuff around to push to another repo and changing credentials and then back.

Reply:
Yeah, if it's complicated, then you can just uncomment the bintray lines and we'll cross our fingers during the next release :crossed_fingers: :sweat_smile: It'd be ideal if we had a second bintray account and can test everything in a fork though, since that would allow us to more easily iterate on and test the CI scripts for any future modifications. Half of the tediousness of CI scripts comes from the fact that it's difficult to test them...

Reply:
maybe merge this with uncommented lines and if there is time left before the release (even in the last few days) this can be worked on ?

Reply:
It wasn't much work to create a separate Bintray account and identical repos, but [I'm getting a 400 on upload](https://github.com/imiric/k6/runs/1241346992), which could be anything... I'll see if I can quickly troubleshoot this without exposing the secrets, but I'd rather ensure this works now than have to do it during a release.

Reply:
you can expose things like the API key for your bintray account and then just reset it from bintray, if that will help with the debugging
and yeah, for a 400 status, :+1: for debugging now and not under pressure after a release

Reply:
OK, so the 400 was caused because the packages in Bintray needed to be manually created. After doing that, and fixing a `GPG_PASSPHRASE` secret name typo in my repo, the packages uploaded fine in [this build](https://github.com/imiric/k6/actions/runs/302022109), and you can see them [here](https://bintray.com/imiric2). I had to disable all tests as I was getting a flaky failure on previous builds... :disappointed: The flakiness is really a major problem that's wasting a lot of time in day-to-day work, and gives off a bad impression to external contributors.

Anyway, I'll clean things up and squash some commits, and push the final version to this PR in a few minutes.

There's also an `add-path` deprecation warning I just now noticed. Will look into it if it's a simple fix.